examples/scenarioIntegrators.py

```diff
@@ -171,17 +171,17 @@ def run(show_plots, integratorCase):
         show_plots (bool): Determines if the script should display plots
```

Comment:
:large_orange_diamond: this is not a hot-fix. Just a regular commit.

Reply:
I fixed the commit message